Update index.md (#5000)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/configure-amr-claims-mapping/main/index.md

#### What you need

* [Okta Developer Edition organization](https://developer.okta.com/signup)

> * An existing OpenID Connect Identity Provider (IdP) that's able to send AMR claims to Okta. This can be another Okta org (org2org) or a third party IdP. If your current Org2Org uses SAML, you'll need to implement a new one with OIDC.

Reviewer: ```suggestion
* An existing OpenID Connect Identity Provider (IdP) that's able to send AMR claims to Okta. This can be another Okta org (org2org) or a third party IdP. If your current org2org uses SAML, you need to create one using OpenID Connect.
```

#### Use an existing org2org app

> You can configure Okta-to-Okta orgs for AMR claims mapping with existing org2org apps (OIDC). If you want to force the IdP Okta org to authenticate, clear the **Disable Force Authentication** checkbox in the existing org2org app:

Reviewer: ```suggestion
You can configure Okta-to-Okta orgs for AMR claims mapping with existing OIDC org2org apps. If you want to force the IdP Okta org to authenticate, clear the **Disable Force Authentication** checkbox in the existing org2org app:
```
